plugins/inputs/redis/redis.go

```diff
@@ -64,6 +103,11 @@ var sampleConfig = `
   ## If no port is specified, 6379 is used
   servers = ["tcp://localhost:6379"]
 
+  ## Optional. Specify redis commands to retrieve values, under key "commands"
+  # command = ["get", "sample-key"]
+  # key = "sample-key-value"
+  # type = "string"
+
```

Comment:
this looks wrong. Don't you need to specify `[[inputs.redis.commands]]` ahead of this?

Reply:
consider naming "key" `field` instead.

Reply:
changes should be copied into readme.

Reply:
This is addressed! Thank you for the catch, let me know if there is anything else needed.